### PR TITLE
Added source_digest instead of source in coveralls api

### LINKED ISSUE #27: source_digest not part of the source file object
As per coveralls api source_digest is required
```json
{
  "service_job_id": "1234567890",
  "service_name": "travis-ci",
  "source_files": [
    {
      "name": "example.rb",
      "source_digest": "asdfasdf1234asfasdf2345",
      "coverage": [null, 1, null]
    },
    {
      "name": "lib/two.rb",
      "source_digest": "asdf1234asdfsdfggsfgd9423",
      "coverage": [null, 1, 0, null]
    }
  ]
}
```
but it seems like `source_digest` is not send with the json_file 
https://github.com/markusn/coveralls-erl/blob/749eb6352afb8a3041933467f435131713bae856/src/coveralls.erl#L198-L203

### PR DESCRIPTION
**Pull request by shamis**
fixes #27

### CODE REVIEW

**Comment by markusn** [comment]
Thanks, will have a look later this week

File: src/coveralls.erl [hunk 1]
```diff
@@ -381,8 +363,13 @@ realpath_and_relative_test_() ->
   {setup,
    fun() -> %% setup
        {ok, Cwd} = file:get_cwd(),
-       Root = string:strip(
-                os:cmd("mktemp -d -t coveralls_tests.XXX"), right, $\n),
+       RandNum = case erlang:function_exported(rand, uniform, 1) of
```
Comment (markusn) [inline src/coveralls.erl]: this seems completely unrelated to source digest, why the change?

Comment (markusn) [reply src/coveralls.erl]: I'd prefer if that was part of a separate PR and that this PR would be squashed into 1 commit :).

**Comment by shamis** [comment]
@markusn, have made the changes please have a look
